composer.json

```diff
@@ -6,6 +6,11 @@
     "require-dev": {
```

Comment:
also should we update this ` "guzzlehttp/guzzle": "^7.7"`?

Reply:
my updates in `composer.lock` is enough to fix the vunerability. But if you still want to update this, I can do the update as well.

Reply:
yes please update it here as well

Reply:
@vigan-abd I updated. Please check! thanks!